docs/docs/en/guides/develop/automate/continuous-integration.md

```diff
@@ -116,14 +116,14 @@ We recommend using `mise use --pin` in your Tuist projects to pin the version of
 
 ## Authentication
 
-When using server-side features such as [cache](/guides/develop/build/cache), you'll need a way to authenticate requests going from your CI workflows to the server. For that, you can generate a project-scoped token by running the following command:
+When using server-side features such as [cache](/en/guides/develop/build/cache), you'll need a way to authenticate requests going from your CI workflows to the server. For that, you can generate a project-scoped token by running the following command:
```

Comment:
Are these links automatically changed to Korean if that's the selected language?

Reply:
> Are these links automatically changed to Korean if that's the selected language?

The Korean version of the document has the links prefixed with `/ko`. Note that there's some room for mistakes here (e.g. forgetting to update the locale of a link), so I'd suggest that we introduce some link linter that fails if we detect those.